example/lib/meeting/meeting_store.dart

```diff
@@ -691,6 +688,20 @@ class MeetingStore extends ChangeNotifier
         peerTracks[index].setAudioLevel(element.audioLevel);
       }
     });
+    if (isPipActive) {
```

Comment:
@Decoder07  does this check PIP active only on Android or on iOS as well?

Reply:
It's for both.